packages/neon-core/src/wallet/Account.ts

```diff
@@ -88,6 +88,20 @@ export class Account implements NeonObject<AccountJSON> {
     });
   }
 
+  public static validateKey(str: string): string | void {

```

Comment:
Correct me if I am wrong but conventionally, validate returns a bool. If the objective is to also provide some information about the string, may I suggest the name `identifyString `instead.

Reply:
can do both with a falsely check with an empty string